### DISCUSSION TITLE
Show each respective output seperately

### DISCUSSION THREAD

**Discussion by ChurroC** [vercel/turborepo#8091] [Ideas]
### Goals

Currently its very difficult to look through the cli and understand what is happening. Especially when you have a backend, frontend, and packages being built, it easily becomes difficult to keep track of what is happening in the terminal. I just tried buns filtering tool and it does wonders for simplifying the cli. It would be great if we could use that idea with turbo to improve caching.

### Non-goals

This isn't meant to change anything the cli shows but just organize it to be more manageable during dev.

### Background

I've been trying to run dev for my whole website for the backend, frontend, and the packages I made. The problem that ends up happening is that once all the cli's start spilling onto the terminal, even with separate colors it's very hard to understand. By implementing seperate sections for each package we can easily traverse the terminal and be able to see what exactly is happening.

Currently with turbo:
![image](https://github.com/vercel/turbo/assets/66274903/97943131-34a4-46a6-b509-3c8f6f0d277c)

With bun filter:
![image](https://github.com/vercel/turbo/assets/66274903/f453c17e-3f82-4d7b-9149-adcde4a04838)

Even with hot reloads it only specifically modified the specific packages terminal.

### Proposal

I would be interested in contributing but I want to know if this is something other would prefer.

**Comment by anthonyshew** [comment]
Try out the new UI in https://github.com/vercel/turbo/discussions/7802. We'd love to hear your feedback!
